pingaccess/resource_pingaccess_load_balancing_strategy.go

```diff
@@ -0,0 +1,125 @@
+package pingaccess
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/iwarapter/pingaccess-sdk-go/pingaccess"
+)
+
+func resourcePingAccessLoadBalancingStrategy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePingAccessLoadBalancingStrategyCreate,
+		Read:   resourcePingAccessLoadBalancingStrategyRead,
+		Update: resourcePingAccessLoadBalancingStrategyUpdate,
+		Delete: resourcePingAccessLoadBalancingStrategyDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: resourcePingAccessLoadBalancingStrategySchema(),
+	}
+}
+
+func resourcePingAccessLoadBalancingStrategySchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		className: &schema.Schema{
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validateClassNameValue,
+		},
+		configuration: &schema.Schema{
+			Type:             schema.TypeString,
+			Required:         true,
+			DiffSuppressFunc: suppressEquivalentConfigurationDiffs,
+		},
+		name: &schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+	}
+}
+
+func resourcePingAccessLoadBalancingStrategyCreate(d *schema.ResourceData, m interface{}) error {
+	svc := m.(*pingaccess.Client).HighAvailability
+	name := d.Get(name).(string)
```

Comment:
This should just be used directly in the LoadBalancingStrategyView definition i.e.
```go
Body: pingaccess.LoadBalancingStrategyView{
	Name: String(d.Get(name).(string)),
```

Reply:
Created `resourcePingAccessLoadBalancingStrategyReadData`